src/Ldap.php

```diff
@@ -700,20 +700,17 @@ public function connect($host = null, $port = null, $useSsl = null, $useStartTls
             throw new Exception\LdapException(null, 'A host parameter is required');
         }
 
-        $useUri = false;
         /* Because ldap_connect doesn't really try to connect, any connect error
          * will actually occur during the ldap_bind call. Therefore, we save the
          * connect string here for reporting it in error handling in bind().
          */
         $hosts = [];
         if (preg_match_all('~ldap(?:i|s)?://~', $host, $hosts, PREG_SET_ORDER) > 0) {
             $this->connectString = $host;
-            $useUri              = true;
             $useSsl              = false;
```

Comment:
@heiglandreas 
I leaved this assignment untouched for now, but should we recognize $useSsl here depending on schema? I mean *ldaps://* should switch $useSsl to true, shouldn't it?

Reply:
That's a good catch! Would you add that?

Reply:
done